src/test/java/com/palantir/docker/compose/AggressiveShutdownStrategyIntegrationTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ */
+
+package com.palantir.docker.compose;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import com.palantir.docker.compose.configuration.ShutdownStrategy;
+import com.palantir.docker.compose.logging.DoNothingLogCollector;
+import org.junit.Test;
+
+public class AggressiveShutdownStrategyIntegrationTest {
+
+    @Test
+    public void shut_down_multiple_containers_immediately() throws Exception {
+        DockerComposeRule rule = DockerComposeRule.builder()
+                .file("src/test/resources/shutdown-strategy.yaml")
+                .logCollector(new DoNothingLogCollector())
+                .retryAttempts(0)
+                .shutdownStrategy(ShutdownStrategy.AGGRESSIVE)
+                .build();
+
+        assertThat(rule.dockerCompose().ps(), is(TestContainerNames.of()));
+
+        rule.before();
```

Comment:
Might be worth asserting that the containers are non-empty, so we can know that we're actually killing things.

I suppose it's also quite hard to test that things are aggressively removed :S